tests/validation/test-class-amp-validation-error-taxonomy.php

```diff
@@ -513,6 +705,38 @@ public function test_add_error_type_clauses_filter() {
 		$this->assertEquals( $initial_clauses, apply_filters( $tested_filter, $initial_clauses, $taxonomies ) );
 	}
 
+	/**
+<<<<<<< HEAD
```

Comment:
It looks like this is still here from resolving the (many) merge conflicts.

Reply:
Fixed with https://github.com/Automattic/amp-wp/pull/1429/commits/cd2010670c8e0ab8e4baa303148612f2c29debde